### ISSUE TITLE
History chapter selection

### ISSUE THREAD

**Issue by antocs** [tachiyomiorg/tachiyomi#7338]
### Steps to reproduce

Try to open a chapter from the History list.

### Expected behavior

It should open it normally.

### Actual behavior

If there are no more chapters after the selected one, it won't open it and give out the "Next chapter not found" error.

### Tachiyomi version

Preview r4489

### Android version

Android 12

### Device

Samsung Galaxy S10

### Other details

If the video is hard to follow:
I try to open Chainsaw Man but it won't because there is no chapter after the last one ("Next chapter not found" error).
Then I try to open Maoujou de Oyasumi and it works because there is one more chapter after the selected one.

https://user-images.githubusercontent.com/22413659/174474568-23b33e14-1e55-463b-bc77-6b81593966de.mp4

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by antocs** [comment]
It works if I set the chapter as unread, so it's intended to not work if I already read everything?

**Comment by AntsyLich** [comment]
> It works if I set the chapter as unread, so it's intended to not work if I already read everything?

It's supposed to continue from where you left off. So if you completely read the chapter and there's none after you can't really continue can you.

**Comment by Soitora** [comment]
As I said on Discord:

> Honestly kinda agree, if you click chapter 123, you expect 123 to open, not it to open 124
> It's a history view, not "current progress" view
> 
> You don't go to your YouTube History to find Episode 100 of a podcast you like watching only for YouTube to deny you because Episode 101 isn't out yet
> 
> That sounds incredibly unintuitive

**Comment by arkon** [comment]
It'll continue where you left off (in the middle of a chapter) or the next available chapter. That's always been the case, so I'm not sure it makes sense to change it at this point.

Tapping the cover opens the manga details.
